recipe/recipe.yaml

```
@@ -1,91 +1,83 @@
-{% set name = "hist" %}
-{% set version = "2.8.1" %}
-{% set python_min = "3.8" %}
+schema_version: 1
 
+context:
+  name: hist
+  version: 2.8.1
+  python_min: 3.8
 
-package:
-  name: {{ name|lower }}-suite
-  version: {{ version }}
+recipe:
+  name: ${{ name|lower }}-suite
+  version: ${{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/hist-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/hist-${{ version }}.tar.gz
   sha256: 7da7c900e2ef6d425793da1a9adac424ebc013a8eabf29b24301f70898218d9d
 
 build:
-  number: 1
+  number: 2
 
 outputs:
-  - name: hist-base
+  - package:
+      name: hist-base
     build:
-      noarch: python
-      entry_points:
-        - hist=hist.classichist:main
       # c.f. https://github.com/conda/grayskull/issues/582#issuecomment-2482370940
       # FIXME: For unknown reasons, use of '{{ PYTHON }} -m pip' fails here
+      noarch: python
       script: python -m pip install . -vv --no-deps --no-build-isolation
-
+      python:
+        entry_points:
+          - hist=hist.classichist:main
     requirements:
       host:
         - pip
-        - python {{ python_min }}
+        - python 3.8.*
         - hatchling
         - hatch-vcs
       run:
         - boost-histogram >=1.3.1,<1.6
         - histoprint >=2.2
         - numpy >=1.14.5
-        - python >={{ python_min }}
+        - python >=${{ python_min }}
         # Actually only required for [py<311] but require for all Python versions so this can be noarch: python
         - typing-extensions
-
-    test:
-      imports:
-        - hist
-        - hist.axis
-      commands:
-        - pip check
-      requires:
-        - python {{ python_min }}
-        - pip
-
-  - name: hist
-
+    tests:
+      - python:
+          imports:
+            - hist
+            - hist.axis
+          pip_check: true
+  - package:
+      name: hist
     build:
       noarch: python
-
     requirements:
       # Satisfy linter but not used
       host:
-        - python {{ python_min }}
-
+        - python 3.8.*
```

```suggestion
        - python ${{ python_min }}.*
```

Bug from https://github.com/conda/conda-recipe-manager/issues/307